tools/wptrunner/wptrunner/executors/actions.py

```diff
@@ -277,6 +277,85 @@ def __call__(self, payload):
         self.logger.debug("Setting SPC transaction mode to %s" % mode)
         return self.protocol.spc_transactions.set_spc_transaction_mode(mode)
 
+class CancelFedCMDialogAction:
+    name = "cancel_fedcm_dialog"
+
+    def __init__(self, logger, protocol):
+        self.logger = logger
+        self.protocol = protocol
+
+    def __call__(self, payload):
+        self.logger.debug("Canceling FedCM dialog")
+        return self.protocol.fedcm.cancel_fedcm_dialog()
+
+class SelectFedCMAccountAction:
+    name = "select_fedcm_account"
+
+    def __init__(self, logger, protocol):
+        self.logger = logger
+        self.protocol = protocol
+
+    def __call__(self, payload):
+        account_index = payload["account_index"]
+        self.logger.debug("Selecting FedCM account of index: %s" % account_index)
```

Comment:
Assuming that a number is passed to the testdriver.js API, `account_index` should be either a Python `int` or `float` now, not a string. But you can avoid this `%s` like this:

```suggestion
        self.logger.debug(f"Selecting FedCM account of index: {account_index}")
```

Reply:
Good point, fixed and thank you!